src/players/youtube.py

```diff
@@ -28,23 +28,18 @@
 ytdl = youtube_dl.YoutubeDL(ytdl_format_options)
 
 
-class YoutubePlayer(discord.PCMVolumeTransformer, BasePlayer, name='youtu'):
+class YoutubePlayer(BasePlayer, domain_name='youtu'):
     def __init__(self, source, *, data, volume=1):
         super().__init__(source, volume)
         self.data = data
         self.title = data.get('title')
         self.url = data.get('url')
 
     @classmethod
-    def from_url(cls, url: str, stream: bool = False) -> discord.PCMVolumeTransformer:
+    def from_url(cls, url: str, stream: bool = False) -> YoutubePlayer:
```

Comment:
Seems like stream using alltime, just use it into `from_url` method, and drop argument